compiler/rustc_target/src/spec/targets/wasm32_wasi_preview1.rs

```diff
@@ -0,0 +1,57 @@
+//! The `wasm32-wasi-preview1` enables compiling to WebAssembly using the first
+//! version of the WASI standard, called "preview1". This version of the
+//! standard was never formally specified and WASI has since evolved to a
+//! "preview2". This target in rustc uses the previous version of the proposal.
+//!
+//! This target uses the syscalls defined at
+//! <https://github.com/WebAssembly/WASI/tree/main/legacy/preview1>.
+//!
+//! Note that this target was historically called `wasm32-wasi` originally and
+//! was since renamed to `wasm32-wasi-preview1` after the preview2 target was
+//! introduced.
```

Comment:
I'll note that this file started as an exact copy of the one from `wasm32_wasi.rs` but I took the liberty of updating this (now very old) comment while I was here.